x/tradeshield/migrations/v8_migration.go

```diff
@@ -0,0 +1,45 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/tradeshield/types"
+)
+
+func (m Migrator) V8Migration(ctx sdk.Context) error {
+	allSpotOrders := m.keeper.GetAllLegacyPendingSpotOrder(ctx)
+	for _, order := range allSpotOrders {
+		m.keeper.RemovePendingSpotOrder(ctx, order.OrderId)
+		m.keeper.SetPendingSpotOrder(ctx, types.SpotOrder{
```

Comment:
Given that some orders might exist with incorrect params, would be nice to double check this will map correctly